crates/relayer-framework/src/tests/relayer_mock/base/types/state.rs

```diff
@@ -0,0 +1,48 @@
+use alloc::string::{String, ToString};
+use std::{collections::HashSet, fmt::Display};
+
+type PortId = String;
+type ChannelId = String;
+type Sequence = u128;
+
+#[derive(Clone, Default, Debug)]
+pub struct State {
```

Comment:
We should also have a `sent_packets` field that records packets that are being sent. We would then modify this state directly during tests, to insert the packets we want to send.